packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -634,6 +636,57 @@ export const RemoteServerFactory = {
         })
     },
 
+    /**
+     * @private
+     */
+    _setupExtensions(app, options) {
+        const extensions = options.mobify?.app?.extensions || []
+        app.__extensions = extensions || []
+
+        let _require
+
+        extensions.forEach((extension) => {
+            const setupServerFilePath = path.join(
+                options.buildDir,
+                'extensions',
+                extension,
+                'setup-server.js'
```

Comment:
I guess we can re-factor the logic in a separate util and re-use it to support both ts and js file extensions.

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/c6480c8f36029ff3770167ca41415db50913957c/packages/pwa-kit-dev/src/ssr/server/build-dev-server.js#L146-L153

Something like:

```
_getSetupServerFilePath(setupServerFilePathBase) {

    if (fs.existsSync(`${setupServerFilePathBase}.ts`)) { 
         filePath = `${setupServerFilePathBase}.ts` 
     } else if (fs.existsSync(`${setupServerFilePathBase}.js`)) { 
         filePath = `${setupServerFilePathBase}.js` 
     } else { 
         return 
     } 
}
    ```

Reply:
The remote server will never see the TS file, because we compile it to JS and upload the JS file as a part of the build.